operations/policies/policies_deployment.rego

```diff
@@ -0,0 +1,55 @@
+package main
+
+# Tests which relate to the expected Deployment manifests
+
+is_rollout_operator_deployment_valid(obj) if {
```

Comment:
I don't think checks like this one are valuable - they become change detector tests rather than alerting us to specific problems. Are there specific things that are important to check here?